docs/current_docs/api/fs-filters.mdx

```diff
@@ -179,6 +206,14 @@ To implement a post-call filter in your Dagger Function, use the `include` and `
 ```typescript file=./snippets/fs-filters/post-call/typescript/index.ts
 ```
 
+</TabItem>
+<TabItem value="Java">
+
+To implement a post-call filter in your Dagger Function, use the `Container.WithDirectoryArguments` class which support `withInclude` and `withExclude` functions when working with `Directory` objects. Here's an example:
+
+```typescript file=./snippets/fs-filters/post-call/typescript/index.ts
+```
```

Comment:
Looks like it's using the typescript snippet in the java tab

Reply:
Fixed 👍